Leanplum/Api/LPDeviceApi.m

```diff
@@ -14,9 +14,10 @@
 
 @implementation LPDeviceApi
 
-+ (void) setDeviceAttributes:(NSString *)deviceId withDeviceAttributes:(NSDictionary *)attributes
-                     success:(void (^)(void))success
-                     failure:(void (^)(NSError *error))failure {
++ (void) setDeviceId:(NSString *)deviceId
+withDeviceAttributes:(NSDictionary *)attributes
+             success:(void (^)(void))success
+             failure:(void (^)(NSError *error))failure {
```

Comment:
why is this change necessary? In old SDK the api is setDeviceAttributes not setDeviceId

Reply:
i think @gg4race , the change is easier to read, cos we do have "withDeviceAttributes" as a param name as well.